specs/Theme.md

```diff
@@ -0,0 +1,130 @@
+Theme API
+===
+
+# Background
+This API's main use is to set the overall theme for WebView2's which are associated with a profile. The options are similar to Edge: match the system default theme, change to light theme, or change to dark theme.
+This sets the theme for WebView2 UI like dialogs, prompts, and context menus by setting the ['prefers-color-scheme'](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) [media feature](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries#media_features). Websites typically query for this setting in order to set CSS themes for light/dark.
+
```

Comment:
If I set the theme to "System", what happens to the prefers-color-scheme media query?

Option 1: We match light/dark based on the user's app color preference (in the Settings app).
Option 2: We match some hard-coded value. (Which one? "system", "light", "dark".)
Option 2a: We match some hard-coded value as in Option 2, unless the user is in a high contrast mode, in which case we report light/dark based on whether the user is in High Contrast Light or High Contrast Dark.
Option 3: We check the user's `COLOR_WINDOW` and infer light/dark based on that.
Option 4: We match nothing (all media queries fail).
Option 5: We match whatever the default Edge profile's color preference is.

Reply:
Option 1 would be the outcome. I can expand on the doc comment to clarify the impact that setting `System` has on prefers-color-scheme

Reply:
I believe "Let the system choose one of the other options for me" is traditionally named "Auto". @MikeHillberg  can double-check me on that.

Reply:
Agreed, 'System' is a little confusing, 'Auto' has more precedent (if not 100%)

Reply:
I think we were looking to match Edge's appearance page naming, which was System (technically System Default). If Auto is the standard however we can rename to that. @david-risney What do you think?

Reply:
Changed to Auto for now since it has more precedent.